pinot-core/src/main/java/org/apache/pinot/core/segment/extracolumn/ExtraColumnContext.java

```diff
@@ -16,14 +16,28 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.core.segment.virtualcolumn;
+package org.apache.pinot.core.segment.extracolumn;
+
+import org.apache.pinot.common.data.FieldSpec;
+
 
 /**
- * Virtual column provider that contains the current segment name.
+ * Context used to pass arguments to the extra column provider.
  */
-public class SegmentNameVirtualColumnProvider extends SingleStringVirtualColumnProvider {
-  @Override
-  protected String getValue(VirtualColumnContext context) {
-    return context.getSegmentName();
+public class ExtraColumnContext {
```

Comment:
SegmentNameVirtualColumnProvider got renamed to ExtraColumnContext?

Reply:
Github somehow misaligned the files.. 
- [VirtualColumnContext](https://github.com/apache/incubator-pinot/blob/master/pinot-core/src/main/java/org/apache/pinot/core/segment/virtualcolumn/VirtualColumnContext.java) -> ExtraColumnContext
- SegmentNameVirtualColumnProvider and HostNameVirtualColumnProvider are removed, and merged to SingleStringExtraColumnProvider